styles/globals.css

```
@@ -325,13 +345,18 @@
   filter: drop-shadow(0 0 4px #0083ef) drop-shadow(0 0 8px #0083ef);
 }
 
+.mind-area-section {
+  corner-shape: superellipse(1.2);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)